src/main/java/graphql/nadel/result/ExecutionResultNode.java

```diff
@@ -81,6 +83,14 @@ public NonNullableFieldWasNullException getNonNullableFieldWasNullException() {
                 .findFirst();
     }
 
+    public Integer getResultNodeCount() {
+        return totalNodeCount.get();
+    }
+
+    public void setTotalNodeCount(AtomicInteger newNodeCount) {
+        totalNodeCount.set(newNodeCount.get());
+    }
```

Comment:
Never take an AtomicInteger as a parameter value (in order to get its value).  Just use `int`

if it looks weird that you have `var.set(param.get())` then your right - its weird